src/lib/sw_engine/tvgSwCommon.h

```diff
@@ -313,6 +313,7 @@ void rleFree(SwRleData* rle);
 void rleReset(SwRleData* rle);
 void rleClipPath(SwRleData *rle, const SwRleData *clip);
 void rleClipRect(SwRleData *rle, const SwBBox* clip);
+void shapeMaskingAlpha (SwShape *compShape, SwRleData *rle, const SwRleData *clip);
```

Comment:
Please append to end of shape prototypes..  (278 - 290 lines)

Reply:
I changed name to rleAlphaMask, still I should move this?